src/main/resources/config/application-prod.yml

```diff
@@ -78,7 +89,7 @@ spring:
 # ===================================================================
 server:
   use-forward-headers: true
-  port: 8080
+  port: ${PORT}
```

Comment:
you can always override with environment variables like `SERVER_PORT`

Reply:
This PORT variable is GCP specific. It has nothing to do with a GiHub environment variable. :smile: